docs/docs/index.md

```diff
@@ -20,6 +20,7 @@ group: nav-right
 **[Interpreters in zeppelin](manual/interpreters.html)**
 
 * [flink](../docs/pleasecontribute.html)
+* [hdfs](./interpreter/hdfs.html)
```

Comment:
Thanks for looking at the change again. The real ones are different from the missing ones (../docs/pleasecontribute.html), below is a larger snippet that should help:

* [flink](../docs/pleasecontribute.html)
* [hdfs](./interpreter/hdfs.html)
* [hive](../docs/pleasecontribute.html)
* [ignite](../docs/pleasecontribute.html)
* [lens](../docs/pleasecontribute.html)
* [md](../docs/pleasecontribute.html)
* [sh](../docs/pleasecontribute.html)
* [spark](./interpreter/spark.html)
* [tajo](../docs/pleasecontribute.html)
* [postgresql, hawq](./interpreter/postgresql.html)
* [geode](./interpreter/geode.html)